radicle-httpd/src/api/v1/projects.rs

```diff
@@ -375,11 +374,12 @@ struct Stats {
     contributors: usize,
 }
 
-fn stats<R: WriteRepository>(repo: &R) -> Result<Stats, Error> {
-    let branches = repo.raw().branches(Some(BranchType::Local))?.count();
-    let (_, head) = repo.head()?;
+fn stats(repo: &Repository) -> Result<Stats, Error> {
+    let branches = repo.branches(Glob::all_heads())?.count();
```

Comment:
Is this equivalent? ie. it won't pick up remotes?

Reply:
correct. It won't pick up remotes.

Reply:
We have a function for this https://github.com/radicle-dev/radicle-git/blob/main/radicle-surf/src/git/repo.rs#L272-L288